ui/app/services/system.js

```diff
@@ -113,37 +113,6 @@ export default class SystemService extends Service {
     return namespaces.length && namespaces.some(namespace => namespace.get('id') !== 'default');
   }
 
-  @computed('namespaces.[]')
```

Comment:
Since `system.cachedNamespace` is set and used elsewhere, what do you think of adding a “declaration” for it, and maybe an explanatory comment?